### ISSUE TITLE
update comment in epoll reactor

### ISSUE THREAD

**Issue by twisted-trac** [twisted/twisted#5440]
|<img alt="benjamin's avatar" src="https://avatars.githubusercontent.com/u/0?s=50" width="50" height="50">| benjamin reported|
|-|-|
|Trac ID|trac#5440|
|Type|enhancement|
|Created|2011-12-26 05:42:06Z|

The comment explains the function of the epoll_create argument to the kernel. This patches updates that.

Attachments:

* [update_comment.diff](https://raw.githubusercontent.com/twisted/twistedmatrix.com-trac-attachments/trunk/ticket/1d3/1d3027412b106008f1a8094d747616d37f4ae1bb/ce3104abc0ee61b82970d35fca766b90bca876a0.diff) (777 bytes) - added by benjamin on 2011-12-26 05:42:25Z - 

<details><summary>Searchable metadata</summary>

```
trac-id__5440 5440
type__enhancement enhancement
reporter__benjamin benjamin
priority__normal normal
milestone__ 
branch__ 
branch_author__ 
status__closed closed
resolution__fixed fixed
component__core core
keywords__ 
time__1324878126000000 1324878126000000
changetime__1325424546000000 1325424546000000
version__None None
owner__exarkun exarkun

```
</details>

**Comment by twisted-trac** [comment]
|[<img alt="exarkun's avatar" src="https://avatars.githubusercontent.com/u/254565?s=50" width="50" height="50">](https://github.com/exarkun)<a name="note_1"></a>|@exarkun set owner to @exarkun|
|-|-|

Thanks!  Looks great.  I'll apply it.

**Comment by twisted-trac** [comment]
|[<img alt="exarkun's avatar" src="https://avatars.githubusercontent.com/u/254565?s=50" width="50" height="50">](https://github.com/exarkun)<a name="note_2"></a>|@exarkun set status to `closed`|
|-|-|

(In [33407]) Update comment about the epoll_create argument to reflect more recent Linux's

Author: benjamin
Reviewer: exarkun
Fixes: [#5440](https://github.com/twisted/twisted/issues/5440)
